packages/pwa/app/utils/site-utils.js

```diff
@@ -59,7 +58,7 @@ export const getSiteByHostname = (hostname) => {
         throw new Error('No site config found. Please check you configuration')
     if (!hostname) return
 
-    const site = JSONPath(`$[?(@.hostnames && @.hostnames.includes('${hostname}'))]`, sites)
+    const site = sites.filter((site) => site.hostnames.includes(hostname))
```

Comment:
Is `hostnames` always guaranteed to be an array?

Reply:
No, it is not a required field actually. Thanks for pointing it out.